exhale/graph.py

```diff
@@ -744,20 +771,26 @@ def toHierarchy(self, classView, level, stream, lastChild=False):
                     anchor=html_link
                 )
 
-                # should always have at least two parts (templates will have more)
-                title_as_link_parts = self.title.split(" ")
-                if self.template_params:
-                    # E.g. 'Template Class Foo'
-                    q_start = 0
-                    q_end   = 2
+                if self.kind != "page":
+                    # should always have at least two parts (templates will have more)
+                    title_as_link_parts = self.title.split(" ")
+                    if self.template_params:
+                        # E.g. 'Template Class Foo'
+                        q_start = 0
+                        q_end   = 2
+                    else:
+                        # E.g. 'Class Foo'
+                        q_start = 0
+                        q_end   = 1
+                    # the qualifier will not be part of the hyperlink (for clarity of
+                    # navigation), the link_title will be
+                    qualifier   = " ".join(title_as_link_parts[q_start:q_end])
+                    link_title  = " ".join(title_as_link_parts[q_end:])
                 else:
-                    # E.g. 'Class Foo'
-                    q_start = 0
-                    q_end   = 1
-                # the qualifier will not be part of the hyperlink (for clarity of
-                # navigation), the link_title will be
-                qualifier   = " ".join(title_as_link_parts[q_start:q_end])
-                link_title  = " ".join(title_as_link_parts[q_end:])
+                    # E.g. 'Foo'
+                    qualifier = ""
```

Comment:
Nice :boom:

Reply:
Haha, yeah. I didn't put a lot of effort in trying to generalize this bit. I just find having a leading `Page` noun somewhat redundant.